src/commands/cmd_json.cc

```diff
@@ -594,6 +594,32 @@ class CommandJsonMGet : public Commander {
   }
 };
 
+class CommandJsonMSet : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    if ((args_.size() - 1) % 3 != 0) {
+      return {Status::RedisExecErr, errWrongNumOfArguments};
+    }
+
+    redis::Json json(svr->storage, conn->GetNamespace());
+
+    std::vector<std::string> user_keys;
+    std::vector<std::string> paths;
+    std::vector<std::string> values;
+    for (size_t i = 0; i < (args_.size() - 1) / 3; i++) {
+      user_keys.emplace_back(args_[i * 3 + 1]);
+      paths.emplace_back(args_[i * 3 + 2]);
+      values.emplace_back(args_[i * 3 + 3]);
+    }
+
+    auto s = json.MSet(user_keys, paths, values);
+    if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
```

Comment:
Can you change those codes to make Sonarcloud happy?
[Sonarcloud report](https://sonarcloud.io/project/issues?resolved=false&pullRequest=2228&id=apache_kvrocks&open=AY65WAx3QLtepdGeR4C8&tab=code)